x/merkledb/intermediate_node_db_test.go

```diff
@@ -125,3 +125,70 @@ func TestIntermediateNodeDB(t *testing.T) {
 	require.NoError(it.Error())
 	require.Equal(added+1, count)
 }
+
+func FuzzIntermediateNodeDBConstructDBKey(f *testing.F) {
+	f.Fuzz(func(
+		t *testing.T,
+		key []byte) {
+
+		require := require.New(t)
+		cacheSize := 200
+		evictionBatchSize := cacheSize
+		baseDB := memdb.New()
+		db := newIntermediateNodeDB(
+			baseDB,
+			&sync.Pool{
+				New: func() interface{} { return make([]byte, 0) },
+			},
+			&mockMetrics{},
+			cacheSize,
+			evictionBatchSize,
+		)
```

Comment:
Does this need to be done for every fuzz test? Seems like it might make more sense to be put outside the fuzz invocation.